src/test/java/pl/ssitarek/carpark/PaymentImplTest.java

```diff
@@ -0,0 +1,64 @@
+package pl.ssitarek.carpark;
+
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.math.BigDecimal;
+
+import static org.junit.Assert.assertEquals;
+
+public class PaymentImplTest {
+
+    private final int DIFF_IN_CENT = 1;
+    private PaymentImpl payment;
+    private BigDecimal toPay;
+
+    @Before
+    public void setUp() {
+        payment = new PaymentImpl();
+        toPay = new BigDecimal(100);
+    }
+
+    // names should be camelCase but PLN, USD is a name of currency
```

Comment:
po co ten komentarz?

Reply:
Jeden approver dopuszcza customerID inny woli customerId. Chodziło mi o to, że to nazwa testPaymentPLN nie wynika z mojego błędu w stosowaniu camelCase